docs/scripts/update-all-docs.sh

```diff
@@ -111,3 +111,27 @@ echo ".. _list_toolchains:" > $VERSION_SPECIFIC_DIR/toolchains.rst
 echo "*(see also* \`\`eb --list-toolchains\`\` *)*" >> $VERSION_SPECIFIC_DIR/toolchains.rst
 echo >> $VERSION_SPECIFIC_DIR/toolchains.rst
 eb --list-toolchains --output-format rst >> $VERSION_SPECIFIC_DIR/toolchains.rst
+
+#  available toolchain options
+echo ".. _avail_toolchain_opts:" > $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+echo >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+echo "*(see also* \`\`eb --avail-toolchain-opts <tcname>\`\` *)*" >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+echo >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+IFS=$'\n'
+
+for i in `eb --list-toolchains | sed 1d | sort`
+do
+    tc=`echo $i | cut -d ':' -f1`
+    echo "- ${tc//[[:blank:]]/}_" >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
```

Comment:
I wouldn't make this a list, since this leads to a very long line on top of the page, cfr. http://boegel-eb.readthedocs.io/en/test/version-specific/toolchain_opts.html

I'd make this a single line instead, like we do on http://boegel-eb.readthedocs.io/en/test/version-specific/generic_easyblocks.html